internal/core/renderer.rs

```diff
@@ -109,7 +109,7 @@ pub trait RendererSealed {
 
     fn set_window_adapter(&self, _window_adapter: &Rc<dyn WindowAdapter>);
 
-    fn resize(&self, _size: crate::api::PhysicalSize) -> Result<(), PlatformError> {
+    fn resize(&self, _size: crate::platform::NonZeroPhysicalSize) -> Result<(), PlatformError> {
```

Comment:
Is it really invalid to have a window size of zero?

For example, imagine a window embedded in another window (a frame)  and it is resized to its minimum size by the user so that it is zero.  

I think IMHO a zero size is a valid size, and that we should resize the renderer appropriately.
Then the renderer should know that there is nothing to render and could destroy the surface or do whatever. But I feel we should still pass an actual size to the renderer::resize.

This is a private API, so this doesn't set it in stone anyway.

Reply:
Yep, you're right - I'll change it back.